addons/account/views/account_unit_test.xml

```diff
@@ -1,62 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
unnecessarily editing the upstream code gives us the chance of merge conflicts we have to sort out manually, first having to dig through the git history to find out why a specific change was made in the first place. So I highly doubt that's worth it